extensions/PhabBugz/lib/Revision.pm

```diff
@@ -166,7 +168,18 @@ sub BUILDARGS {
 #           "view": "public",
 #           "edit": "admin"
 #         },
-#         "bugzilla.bug-id": "1154784"
+#         "bugzilla.bug-id": "1154784",
+#         "uplift.request": {
+#             "User impact if declined": "Some impact.",
+#             "Code covered by automated testing": true,
+#             "Fix verified in Nightly": true,
+#             "Needs manual QE test": false,
+#             "Steps to reproduce for manual QE testing": "none",
+#             "Risk associated with taking this patch": "low",
+#             "Explanation of risk level": "Affects only nondefault configurations.",
+#             "String changes made/needed": "none",
+#             "Is Android affected?": true
+#         }
```

Comment:
Note: I'm going to change this slightly because I would like to include the markdown comment already rendered.